scripts/pkg/install.sh

```diff
@@ -30,6 +30,9 @@ fi
 MACHINE_TYPE=`uname -m`
 if [[ $MACHINE_TYPE == "x86_64" ]]; then
   ARCH='x64'
+elif [[ $OSTYPE == "darwin"* ]] && [[ $MACHINE_TYPE == "arm64" ]]; then
```

Comment:
Can you confirm wether we need to adjust this util:
https://github.com/serverless/serverless/blob/d52526bb6059ce20eba341c29ad5a2373c238624/lib/utils/standalone.js#L15-L25


I assume that not, as I take that binary as run through rosetta will internally assume we run things on x84 architecture, but if it'll detect it's ARM, then it'll generate wrong link for binary update

Reply:
Great catch, I've tested and actually, it reports `arm64` - I've adjusted and tested that update works correctly on M1 macs too